docs/guides/cheatsheet/repl.rst

```diff
@@ -11,14 +11,20 @@ a semicolon and press "ENTER".
     db> select 5;
     {5}
 
-Alternatively, you can run the query without a semicolon with ⌥-Enter on macOS
+Alternatively, you can run the query without a semicolon with esc+return on macOS
 or Alt-Enter on Windows/Linux.
```

Comment:
```suggestion
Alternatively, you can run the query without a semicolon by hitting an `ESC`
key immediately followed by `RETURN` on macOS, or Alt-Enter on Windows/Linux.
```

Reply:
This works even if you hold down the `esc` key